### PR TITLE
Fix "You Ran out of Credits - hours"

### PR DESCRIPTION
**Pull request by 1day2die**

### CODE REVIEW

**Comment by AVMG20** [comment]
what's the fix for what? unclear title / description

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -48,6 +48,30 @@ public function getTimeLeftBoxBackground(float $days)
...
+    *
+    * @return string
+    */
+    public function getTimeLeftBoxUnit(float $days, float $hours)
+    {
+        if ($days < 1)
+        {
+            if ($hours < 1)
+            {
+                return;
```
Comment (AVMG20) [inline app/Http/Controllers/HomeController.php:65]: don't return void if your return docblock says string
can also return null and make return type !string
this code can be simplified

**Comment by 1day2die** [comment]
> what's the fix for what? unclear title / description

please check the linked issue.

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -89,7 +113,7 @@ public function index(Request $request)
 
             $bg = $this->getTimeLeftBoxBackground($days);
             $boxText = $this->getTimeLeftBoxText($days, $hours);
-            $unit = $days < 1 ? 'hours' : 'days';
+            $unit = $this->getTimeLeftBoxUnit($days, $hours);
```
Comment (AVMG20) [inline app/Http/Controllers/HomeController.php:116]: shorthand = return $days < 1 ? ($hours < 1 ? null : "hours") : "days";

**Review by AVMG20 — CHANGES_REQUESTED** [review]

**Pull request merged**